controllers/auth/loginController.js

```diff
@@ -32,7 +32,7 @@ class LoginController {
     }
     User.findOne({
       where: { email },
-      attributes: ["email", "password", "username", "id"]
+      attributes: ["email", "password", "username", "id", "roleId"]
     })
       .then(result => {
```

Comment:
@niomwungeri-fabrice What happens if the query fails? I think you also need to use catch to handle any error from the promise or you can just use async/await

Reply:
@oesukam , i have addressed the issue